cmd/juju/commands/main.go

```diff
@@ -63,13 +63,51 @@ func init() {
 //  Main
 
 var jujuDoc = `
-Juju provides easy, intelligent application orchestration on top of cloud
-infrastructure providers such as Amazon EC2, MaaS, OpenStack, Windows, Azure,
-or your local machine.
+Juju provides easy, intelligent application orchestration on top of Kubernetes,
+cloud infrastructure providers such as Amazon, Google, Microsoft, Openstack,
+MAAS (and more!), or even on your local machine via LXD.
 
-See https://discourse.jujucharms.com/ for ideas, documentation and FAQ.
+See https://juju.is for getting started tutorials and additional documentation.
 `
 
+var usageHelp = `
+Juju provides easy, intelligent application orchestration on top of Kubernetes,
+cloud infrastructure providers such as Amazon, Google, Microsoft, Openstack,
+MAAS (and more!), or even your local machine via LXD.
+
+See https://juju.is for getting started tutorials and additional documentation.
+
+Starter commands:
+
+    add-k8s             Adds a k8s endpoint and credential to Juju.
```

Comment:
I would probably put add-k8s next to add-cloud. Especially with the work to bootstrap to "known kubernetes" so you wouldn't have to always start with the add step.

Reply:
Moved it to the bottom of the list as suggested.